win32ss/gdi/ntgdi/freetype.c

```diff
@@ -3003,7 +3005,7 @@ IntRequestFontSize(PDC dc, FT_Face face, LONG Width, LONG Height)
         /* case (A): lfHeight is negative: the character height */
         Height = -Height;
     }
-    else if (Height > 0)
+    else if (Height > 0 && face->height > 0)
```

Comment:
Do we cover everything now? aka: Height > 0 but face->height <= 0 (if it's possible at all)?

Reply:
face->height <= 0 means an illegal font.

Reply:
face->height <= 0 means an illegal font.

Reply:
So this is handled before/elsewhere?

Reply:
I cannot consider the height when the font was invalid. What to do?

Reply:
I added "ASSERT(face->height > 0);".